doc/calibrations/wave_calib.rst

```diff
@@ -278,36 +278,33 @@ In general, the approach is:
     #. Fit a 2D solution to the lines using the order number as a basis
 
     #. Reject orders where the RMS of the fit (measured in binned pixels)
-       exceeds ``rms_threshold``
+       exceeds a certain threshold set by the user
```

Comment:
Even though it's only a few lines down, I would add a link here that directs to description of how the user sets the threshold.

Reply:
Done